src/Stan/Analysis/Analyser.hs

```diff
@@ -30,19 +30,13 @@ import qualified Data.Set as Set
 import qualified Slist as S
 
 
-{- | Contains analyser function to run on 'HieFile's.
+{- | Create analysing function for 'Inspection' by pattern-matching
+over 'InspectionAnalysis'.
 -}
-data Analyser = Analyser
-    { analyserInspectionId :: !(Id Inspection)
-    , analyserFunction     :: !(HieFile -> Observations)
-    }
```

Comment:
For now, there is no need in the `Analyser` data type, but we may want to return it in the future when we need it.